### DISCUSSION TITLE
Display version of applications in application dashboard

### DISCUSSION THREAD

**Discussion by midestefanis** [argoproj/argo-cd#9021] [Q&A]
How can I see the versions of the microservices in the application dashboard?

**Comment by pasha-codefresh** [comment]
You can see all your images in an application details 
https://cd.apps.argoproj.io/applications/guestbook?view=tree&resource=&node=argoproj.io%2FApplication%2Fargocd%2Fguestbook%2F0

As an alternative solution you can try codefresh gitops dashboard for solve this problem
https://codefresh.io/docs/docs/ci-cd-guides/gitops-deployments/
